sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentStatusDetail.cs

```diff
@@ -31,5 +31,11 @@ public partial class DocumentStatusDetail
         /// <summary> Date time in which the Document's status has been updated. </summary>
         [CodeGenMember("LastActionDateTimeUtc")]
         public DateTimeOffset LastModified { get; }
+
+        /// <summary> Returns true if the document translation is completed. </summary>
+        public bool HasCompleted => Status == TranslationStatus.Succeeded
```

Comment:
@kristapratico will this make sense for Python too?

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/17369